src/pages/Transaction/TransactionCellList/styled.tsx

```diff
@@ -4,7 +4,7 @@ export const TransactionCellListTitlePanel = styled.div`
   display: flex;
   flex-direction: column;
 
-  @media (width <= 750px) {
+  @media (max-width: 750px) {
```

Comment:
The usage of `<=` was introduced by https://github.com/Magickbase/ckb-explorer-frontend/pull/26/files#diff-54b658d530f49987154dab3d72ae3685df6309f7d4eb0152c2fc637cf67ce4f5R61 and it's valid in media query.

I think the problem is caused by the intersection of `<= 750px` and `>= 750px`

Reply:
Got it, media query level 4 is supported since Chrome 104.

Can you estimate the effect that the new syntax introduced @WhiteMinds 

Ref: https://developer.chrome.com/blog/media-query-range-syntax/

Reply:
It's about 83% now, maybe the new syntax should be reverted

Ref: https://caniuse.com/css-media-range-syntax